### DISCUSSION TITLE
Recommneded SSL server certificate provider ?

### DISCUSSION THREAD

**Discussion by acrolink** [strongswan/strongswan#735] [Q&A]
Any recommended authority to issue the server's certificate for the ikev2 VPN protocol? Any special requirements to pay attention to when ordering a certificate? Or is is simply sufficient to use custom local CA?

**Comment by tobiasbrunner** [comment]
Really depends on your requirements and deployment scheme. But [Let's Encrypt](https://letsencrypt.org/) is free and widely trusted.
